deploy/deploy.sh

```diff
@@ -1,34 +1,20 @@
 #! /bin/bash
 
-# Modify THE WANDB_ENTITY value in `environment.sh` (line 16) before running this script.
+# Modify DH_DEPLOY_KEY value in `environment.sh` (line 19) before running this script.
 #
 # Deploys the crypto-real-time-inference application onto a Ubuntu or Debian-based
 # server.
 # 
 # Note that each script called is idempotent.
-# 
-# Args (positional)
-#   - 1) MODELS: selection value for "all", "nn", or "ml"
-
-
-# Validate passed in MODELS value
-if [ -n "$1" ]
-then
-  echo "Using $1 as MODELS selection 🟢"
-  MODELS=$1
-else
-echo "Using default 'all' as MODELS selection"
-  MODELS="all"
-fi
 
 # Setup the deployment environment
 bash deploy/setup_deploy_deb.sh
 
-# Set environment variables for model downloads
+# Set environment variables for deploy
 source ./deploy/environment.sh
 
-# Download models on disk
-poetry run python deploy/download_models_from_wandb.py -s $MODELS
+# Spin up the Prediction server
```

Comment:
What are the consequences of running `docker compose -f deploy/prediction-docker-compose.yml up --detach --pull predict` twice in a row? As of right now, there are no checks in place, in this file or in `deploy_container.sh`, that would stop the command from running if a container is already running. Is the command idempotent?

Reply:
The command `docker compose -f deploy/prediction-docker-compose.yml up --detach --pull predict` is idempotent and if a container (i.e. the service `predict`) is already running it will simply confirm this and take no action.

This command also checks if the image on disk is up to date with the latest image in the Docker Hub repository. If this is true, it takes no action; if this is false, it will pull the latest image from Docker Hub and spin up a new container.